### ISSUE TITLE
Softmax and Triplet loss

### ISSUE THREAD

**Issue by hazemahmed45** [timesler/facenet-pytorch#73]
i tried to use the finetuning notebook, i trained the resnet model using softmax cross entropy for the first trial and with a triplet loss for the second both gived false embeddings. i changed the face detector method instead if mtcnn i used face alignment to crop the faces in case there is a side face the mtcnn would not recognize it. please confirm with me, does the face recognition model relate with the accuracy of the model?

**Comment by ctivanovich** [comment]
Yeah, I am trying to use this for classifying groups of people, and I looked at the final layer given classification and saw that it is just outputting a linear layer. Shouldn't it be applying a softmax function on that output, for multinomial classification?
